Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestRunIndexViewController.swift

```diff
@@ -397,7 +397,7 @@ extension TestRunIndexViewController: RestPutDelegate {
         if responseCode == 200 {
             self.testRunDelegate.didUpdateTestRun()
             DispatchQueue.main.async {
-                self.testRunDelegate.removeUpdatedItemFromTable()
+                self.testRunDelegate.removeUpdatedItemFromTable(status: self.testRun.testStatus)
```

Comment:
Since the status of the run is not set for runs with steps (it is a derived value that the server sets) this will never change from whatever it was when the run was initially loaded. We either need to calculate the status ourselves for a run with steps, or we need to load the cycle again in the TestListVC.